packages/topgrade-bin/topgrade-bin.pacscript

```diff
@@ -1,9 +1,10 @@
 name="topgrade-bin"
 pkgname="topgrade"
-version="7.1.0"
+version="8.2.0"
 url="https://github.com/r-darwish/topgrade/releases/download/v${version}/topgrade-v${version}-x86_64-unknown-linux-gnu.tar.gz"
 description="Upgrades everything on your system"
-hash="f2f61d5833030ff545dedd42a65d7d75e510017de5748da3dd6a9bd6fb32ff52"
+hash="54fe60ef70b21b34c50c0d342ec120aff3a9522ef44a9737f42d5700aed7a1c3"
+breaks="${pkgname}-deb ${pkgname}-app ${pkgname}-git"
```

Comment:
```suggestion
breaks="${pkgname} ${pkgname}-deb ${pkgname}-app ${pkgname}-git"
```

Reply:
Couldn’t this be automated?

Reply:
I guess, if we could set up a bot

Reply:
I mean pacstall could always consider the packages that have the same name but ends with -deb -git -app, excluding the current one, to break the package

Reply:
Yeah this line will be automatically appended in `2.0.0`, the boilerplate `-deb`,`-app`,`-git`, `pkgname` (the suffix will depend on the `name`, a `-deb` pacscript won't have `${pkgname}-deb` as breaks) would be automatically set to break the package.

Any additional packages listed in the `breaks` variable would simply be appended to the auto generated break list.

Reply:
Great!